content/docs/start/list-of-operations.mdx

```diff
@@ -482,3 +488,114 @@ Possible errors:
+
+| Parameters | Type | Description |
+| --- | --- | --- |
+| Asset | asset | Asset that will be held in the ClaimableBalanceEntry. |
+| Amount | integer | Amount of `asset` stored in the ClaimableBalanceEntry. |
+| Claimants | list of claimants | List of Claimants (account address and ClaimPredicate pair) that can claim this ClaimableBalanceEntry. |
+
+Possible errors:
+
+| Error | Code | Description |
+| --- | --- | --- |
+| CREATE_CLAIMABLE_BALANCE_MALFORMED | -1 | The input to this operation is invalid. |
+| CREATE_CLAIMABLE_BALANCE_LOW_RESERVE | -2 | The account creating this entry does not have enough XLM to satisfy the minimum XLM reserve increase caused by adding a ClaimableBalanceEntry. For every claimant in the list, the minimum amount of XLM this account must hold will increase by baseReserve. |
+| CREATE_CLAIMABLE_BALANCE_NO_TRUST | -3 | The source account does not trust the issuer of the asset it is trying to include in the ClaimableBalanceEntry. |
+| CREATE_CLAIMABLE_BALANCE_NOT_AUTHORIZED | -4 | The source account is not authorized to transfer this asset. |
+| CREATE_CLAIMABLE_BALANCE_UNDERFUNDED | -5 | The source account does not have enough funds to transfer `amount` of this asset to the ClaimableBalanceEntry. |
+
+## Claim Claimable Balance
+
+Claims a ClaimableBalanceEntry and adds the amount of asset on the entry to the source account.
+
+Threshold: Low
+
+Result: `ClaimClaimableBalanceResult`
+
+| Parameters | Type | Description |
+| --- | --- | --- |
+| BalanceID | claimableBalanceID | BalanceID on the ClaimableBalanceEntry that the source account is claiming. The balanceID can be retrieved from a succesful `CreateClaimableBalanceResult`. |
+
+Possible errors:
+
+| Error | Code | Description |
+| --- | --- | --- |
+| CLAIM_CLAIMABLE_BALANCE_DOES_NOT_EXIST | -1 | There is no existing ClaimableBalanceEntry that matches the input BalanceID. |
+| CLAIM_CLAIMABLE_BALANCE_CANNOT_CLAIM | -2 | There is no claimant that matches the source account, or the claimants predicate is not satisfied. |
+| CLAIM_CLAIMABLE_BALANCE_LINE_FULL | -3 | The account claiming the ClaimableBalanceEntry does not have sufficient limits to receive amount of the asset and still satisfy its buying liabilities. |
+| CLAIM_CLAIMABLE_BALANCE_NO_TRUST | -4 | The source account does not trust the issuer of the asset it is trying to claim in the ClaimableBalanceEntry. |
+| CLAIM_CLAIMABLE_BALANCE_NOT_AUTHORIZED | -5 | The source account is not authorized to claim the asset in the ClaimableBalanceEntry. |
+
+## Begin Sponsoring Future Reserves
+
+Establishes the is-sponsoring-future-reserves-for relationship between the source account and sponsoredID. See [Sponsored Reserves](../glossary/sponsored-reserves.mdx) for more information.
+
+There must be a corresponding [end sponsoring future reserves](#end-sponsoring-future-reserves) operation in the same transaction to end the is-sponsoring-future-reserves-for relationship. The transaction will fail otherwise.
```

Comment:
Might be worth noting that you get `txBAD_SPONSORSHIP` otherwise? Not sure.

Reply:
Good point. Done.